tools/windows-node-installer/pkg/client/openshift.go

```diff
@@ -68,7 +69,7 @@ func (o *OpenShift) GetCloudProvider() (*v1.PlatformStatus, error) {
 
 // getInfrastructure returns the information of current Infrastructure referred by the OpenShift client or an error.
 func (o *OpenShift) getInfrastructure() (*v1.Infrastructure, error) {
-	infra, err := o.Client.ConfigV1().Infrastructures().Get("cluster", metav1.GetOptions{})
+	infra, err := o.Client.ConfigV1().Infrastructures().Get(context.TODO(), "cluster", metav1.GetOptions{})
```

Comment:
Is it best practice to use `context.TODO()`?

Reply:
The [context](https://golang.org/pkg/context/#pkg-overview) pkg overview gives guidance on this.